lerna.json

```diff
@@ -1,7 +1,5 @@
 {
   "packages": ["packages/*"],
```

Comment:
This should be removed. In the 7.0.0 changelog I noticed it says that the package manager workspace config will be used by default, so this field is now an override of the npm workspaces settings

Reply:
I am not (too) ashamed to admit it took me ~15 minutes to figure out I just needed to remove the `./` from the `workspaces` field.